dotcom-rendering/makefile

```diff
@@ -30,24 +30,23 @@ deploy:
 
 # prod #########################################
 
-build: export NODE_ENV=production
-build: clean-dist install gen-dotenv
+build: clean-dist install
 	$(call log, "building production bundles")
-	@webpack --config ./scripts/webpack/webpack.config.js --progress
+	@NODE_ENV=production webpack --config ./scripts/webpack/webpack.config.js --progress
 
-build-ci: export NODE_ENV=production
-build-ci: clean-dist install gen-dotenv
+build-ci: clean-dist install
 	$(call log, "building production bundles")
-	@SKIP_LEGACY=true webpack --config ./scripts/webpack/webpack.config.js
+	@NODE_ENV=production SKIP_LEGACY=true webpack --config ./scripts/webpack/webpack.config.js
```

Comment:
We’re doing a bit of mix-and-match here. Do we want to settle on one approach?

Either `export NODE_ENV=production` or `@NODE_ENV=production`.

Reply:
Running `export`  exports the value to all child processes created from that shell, which is why you would run it at the beginning of the step and it will percolate down.

Having `NODE_ENV other/stuff/here` just sets it for that process.

The `@` suppresses the echoing of the command:
https://stackoverflow.com/questions/9967105/suppress-echo-of-command-invocation-in-makefile

Reply:
Yea the specific reason for this is if we had node_env set to production in CI it would always fail

Reply:
However I might go back and try and find a tidier way to do this so that it's more clear thats why it works that way!